### PR TITLE
Update README and Quay URLs

### PR DESCRIPTION
**Pull request by andykrohg**

### CODE REVIEW

File: tekton/workshop/content/lab16.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ We'll make use of the `pipeline` `ServiceAccount` in the `%username%-cicd` proje
 Now your upper **Terminal** window is in a shell within the skopeo `Pod`'s container. First, we'll login to Quay. Enter your workshop password when prompted:
 
 ```execute
-skopeo login quayecosystem-quay.quay-enterprise.svc.cluster.local:80 --username=%username% --tls-verify=false --authfile /tmp/auth.json
+skopeo login quayecosystem-quay.devsecops.svc.cluster.local:80 --username=%username% --tls-verify=false --authfile /tmp/auth.json
```
Comment (jharmison-redhat) [inline tekton/workshop/content/lab16.md:39]: needs more delete

File: tekton/workshop/content/lab16.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ skopeo login image-registry.openshift-image-registry.svc.cluster.local:5000 --au
 Finally, let's run a `skopeo copy` to push our `tekton-tasks` image into Quay.
 
 ```execute
-skopeo copy docker://image-registry.openshift-image-registry.svc.cluster.local:5000/%username%-dev/tekton-tasks:latest docker://quayecosystem-quay.quay-enterprise.svc.cluster.local:80/%username%/tekton-tasks:quay1 --src-tls-verify=false --dest-tls-verify=false --authfile /tmp/auth.json
+skopeo copy docker://image-registry.openshift-image-registry.svc.cluster.local:5000/%username%-dev/tekton-tasks:latest docker://quayecosystem-quay.devsecops.svc.cluster.local:80/%username%/tekton-tasks:quay1 --src-tls-verify=false --dest-tls-verify=false --authfile /tmp/auth.json
```
Comment (jharmison-redhat) [inline tekton/workshop/content/lab16.md:51]: needs more delete

File: tekton/workshop/content/lab16.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ spec:
         - copy 
         - --debug
         - docker://image-registry.openshift-image-registry.svc.cluster.local:5000/%username%-dev/tekton-tasks:latest  
-        - docker://quayecosystem-quay.quay-enterprise.svc.cluster.local:80/%username%/tekton-tasks:quay
+        - docker://quayecosystem-quay.devsecops.svc.cluster.local:80/%username%/tekton-tasks:quay
```
Comment (jharmison-redhat) [inline tekton/workshop/content/lab16.md:88]: still needs more delete

File: tekton/workshop/content/lab16.md [hunk 4]
```diff
@@ -125,7 +125,7 @@ spec:
     args:
       - copy 
       - docker://image-registry.openshift-image-registry.svc.cluster.local:5000/\$(params.source_image)
-      - docker://quayecosystem-quay.quay-enterprise.svc.cluster.local:80/\$(params.target_image)
+      - docker://quayecosystem-quay.devsecops.svc.cluster.local:80/\$(params.target_image)
```
Comment (jharmison-redhat) [inline tekton/workshop/content/lab16.md:128]: [needs more delete](https://www.youtube.com/watch?v=sAn7baRbhx4)

File: tekton/workshop/content/lab18.md [hunk 5]
```diff
@@ -474,7 +474,7 @@ spec:
     args:
       - copy 
       - docker://image-registry.openshift-image-registry.svc.cluster.local:5000/$(params.source_image
-      - docker://quayecosystem-quay.quay-enterprise.svc.cluster.local:80/$(params.target_image)
+      - docker://quayecosystem-quay.devsecops.svc.cluster.local:80/$(params.target_image)
```
Comment (jharmison-redhat) [inline tekton/workshop/content/lab18.md:477]: I know you won't believe me if I told you.

**Review by jharmison-redhat — CHANGES_REQUESTED** [review]
You had one complete deletion, but `sed` may have failed you.

**Comment by andykrohg** [comment]
@jharmison-redhat what else needs to be deleted? The Quay CR is still named `quayecosystem` in the `devsecops` namespace, so the service address looks correct to me

File: jenkins/workshop/content/lab17.md [hunk 6]
```diff
@@ -60,7 +60,7 @@ In your pipeline, add the Vulnerability Scan Stage after the Build Image Stage.
     stage('Clair Container Vulnerability Scan') {
       steps {
             sh "oc login -u $ocuser -p $ocpass --insecure-skip-tls-verify https://$ocp 2>&1"
-            sh 'skopeo --debug copy --src-creds="$(oc whoami)":"$(oc whoami -t)" --src-tls-verify=false --dest-tls-verify=false' + " --dest-creds=$quayuser:$quaypass docker://docker-registry.default.svc:5000/cicd-$ocuser/jboss-eap70-openshift:1.5 docker://quay-enterprise-quay-enterprise.apps.$ocp/$quayuser/$quayrepo:1.5"
+            sh 'skopeo --debug copy --src-creds="$(oc whoami)":"$(oc whoami -t)" --src-tls-verify=false --dest-tls-verify=false' + " --dest-creds=$quayuser:$quaypass docker://docker-registry.default.svc:5000/cicd-$ocuser/jboss-eap70-openshift:1.5 docker://quay.apps.$ocp/$quayuser/$quayrepo:1.5"
         }
```
Comment (andykrohg) [inline jenkins/workshop/content/lab17.md:64]: This one uses the route, not the service

Comment (jharmison-redhat) [reply jenkins/workshop/content/lab17.md:64]: I think I saw this one first and then my mind was set on that for all of the others...

**Review by jharmison-redhat — APPROVED** [review]

**Pull request merged**